include/exadg/acoustic_conservation_equations/user_interface/field_functions.h

```diff
@@ -40,6 +40,12 @@ struct FieldFunctions
    * beginning of the simulation.
    */
   std::shared_ptr<dealii::Function<dim>> initial_solution_velocity;
+
+  /*
+   * The function right_hand_side is used to evaluate the acoustic source term on the right-hand
+   * side of the mass conservation equation of the acoustic conservation equations.
+   */
+  std::shared_ptr<dealii::Function<dim>> right_hand_side;
```

Comment:
Somewhat related to this comment https://github.com/exadg/exadg/pull/619/files#r1426708752, this is not fully clear to me. It should be stated explicitly which unknowns/dofs (velocity vs. pressure) this term operates on.

The documentation should also be clear about which formulation we consider the strong form with right-hand side "f" and whether this right-hand side "f" applies `c*c` internally or needs to be scaled externally.

Reply:
I will extend the documentation once we decided how to proceed.